### ISSUE TITLE
`#include <QSet>` is missing

### ISSUE THREAD

**Issue by cubimon** [ros-visualization/rqt_image_view#29]
`#include <QSet>` is missing in `image_view.cpp`, which results in a compilation error on the arch linux aur, [here is the issue](https://github.com/ros-melodic-arch/ros-melodic-rqt-image-view/issues/1). Do you want a PR or do you solve it by yourself?

**Comment by greghab** [comment]
I added the include statement, but I'm not sure how to test. I'm using the AUR helper `trizen` to build the package, and it only lets me edit the pkgbuild, not the source files themselves.

**Comment by cubimon** [comment]
You may change the source in the PKGBUILD to your fork

**Comment by cubimon** [comment]
In the best case your pr is accepted and you don't have to change anything

**Comment by greghab** [comment]
Yeah, but the source in the pkgfile is a .tar.gz. My edits are raw source code in a directory. So I suppose I would have to create a new .tar.gz file from the new source files, and then point it to that in the pkgbuild?

**Comment by greghab** [comment]
I did the above: created a new .tar.gz from the new source, and update the sha256, and it compiles fine. I can also attach the new .tar.gz and sha256 sum, or I'm not sure if its created by some build server.

**Comment by cubimon** [comment]
Alternatively:
```diff
diff --git a/PKGBUILD b/PKGBUILD
index b708589..0034d1e 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -40,9 +40,9 @@ depends=(
 	${ros_depends[@]}
 )
 
-_dir="rqt_image_view-${pkgver}"
-source=("${pkgname}-${pkgver}.tar.gz"::"https://github.com/ros-visualization/rqt_image_view/archive/${pkgver}.tar.gz")
-sha256sums=('e24e26dc98404966cc827a4e576a3c7465d82076095d1831eb9da49d3a97c4d5')
+_dir="rqt_image_view"
+source=("git+https://github.com/greghab/rqt_image_view")
+sha256sums=('SKIP')
 
 build() {
 	# Use ROS environment variables.

```

**Comment by dirk-thomas** [comment]
Resolved by #31.
